docs/unique/i3-config-validation-methods.md

```
@@ -233,54 +233,54 @@ That way you can CI-check your dotfiles before deploying them.
 <!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
 ## Related content
-- [shared-package-layout-clarification](shared-package-layout-clarification.md)
-- [Local-Only-LLM-Workflow](local-only-llm-workflow.md)
-- [Chroma Toolkit Consolidation Plan](chroma-toolkit-consolidation-plan.md)
-- [Migrate to Provider-Tenant Architecture](migrate-to-provider-tenant-architecture.md)
-- [Polyglot S-expr Bridge: Python-JS-Lisp Interop](polyglot-s-expr-bridge-python-js-lisp-interop.md)
-- [aionian-circuit-math](aionian-circuit-math.md)
-- [Dynamic Context Model for Web Components](dynamic-context-model-for-web-components.md)
-- [Post-Linguistic Transhuman Design Frameworks](post-linguistic-transhuman-design-frameworks.md)
-- [Promethean Infrastructure Setup](promethean-infrastructure-setup.md)
+- [[shared-package-layout-clarification]]
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)